src/components/common/Dialog/dialogs/RejestracjaEmail/RejestracjaEmail.js

```diff
@@ -0,0 +1,87 @@
+import { TextField, Button } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core';
+import { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { openDialog, FormType } from '../../store/dialogSlice';
+
+const useStyles = makeStyles({
+  field: {
+    marginTop: 15,
+    marginBottom: 20,
+  },
+});
+
+export const RejestracjaEmail = () => {
+  const dispatch = useDispatch();
+
+  const classes = useStyles();
+
+  const [imie, setImie] = useState('');
+  const [nazwisko, setNazwisko] = useState('');
+  const [email, setEmail] = useState('');
+  const [haslo, setHaslo] = useState('');
+
+  return (
+    <>
+      <TextField
+        className={classes.field}
+        id="outlined-basic"
+        label="Imię"
+        variant="outlined"
+        required
+        fullWidth
+        error={imie ? false : true}
```

Comment:
Tutaj możesz pomyśleć nad tym żeby error pokazywał się tylko, jeśli imie jest np krótsze niż kilka znaków lub inny warunek. Np(imie? false: (imie.trim().length <5)). Natomiast to co napisałeś można zrobić np. Boolean(imie)

Reply:
poprawione. Dodałem też do hasła warunek, żeby było 5 znaków

Reply:
Można dodać też do pozostałych pól